config-service/kubernetes.yaml

```diff
@@ -0,0 +1,90 @@
+---
+apiVersion: v1
+kind: Namespace
+metadata:
+  labels:
+    pod-security.kubernetes.io/audit: baseline
+    pod-security.kubernetes.io/audit-version: latest
+    pod-security.kubernetes.io/enforce: baseline
+    pod-security.kubernetes.io/enforce-version: latest
+    pod-security.kubernetes.io/warn: baseline
+    pod-security.kubernetes.io/warn-version: latest
+  name: daqconfig
+---
+# You must still deploy mongodb with its manifests from upstream
+# and create a secret called mongodb-svcbind-root with the required keys
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app.kubernetes.io/app: config-utility
+    app.kubernetes.io/component: config-utility
+  name: config-utility
+  namespace: daqconfig
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app.kubernetes.io/app: config-utility
+      app.kubernetes.io/component: config-utility
+  template:
+    metadata:
+      labels:
+        app.kubernetes.io/app: config-utility
+        app.kubernetes.io/component: config-utility
+    spec:
+      containers:
+      - env:
+        - name: MONGO_HOST
+          valueFrom:
+            secretKeyRef:
+              key: host
+              name: mongodb-svcbind-root
+        - name: MONGO_PORT
+          valueFrom:
+            secretKeyRef:
+              key: port
+              name: mongodb-svcbind-root
+        - name: MONGO_USER
+          valueFrom:
+            secretKeyRef:
+              key: username
+              name: mongodb-svcbind-root
+        - name: MONGO_PASS
+          valueFrom:
+            secretKeyRef:
+              key: password
+              name: mongodb-svcbind-root
+        image: docker.io/plaso/daqconfig-service:latest
```

Comment:
Ouch, that should really not be using my image.

Reply:
I agree, perhaps we can get @dingp to help put this into the dune github container registry?

Reply:
https://github.com/DUNE-DAQ/daq-docker/blob/develop/docs/migrate_to_github_image_registry.md for the documentation

Reply:
That doc suggests we need a DUNE Github Org Admin to help.

Reply:
Now runs out of github container registry.